gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -1264,3 +1354,76 @@ def get_network_function_device_config_info(self, device_data,
                 }
             ]
         }
+
+    @_set_network_handler
+    def get_create_network_function_device_config_info(self, device_data,
+                                                       network_handler=None):
+        """ Get the configuration information for NFD
+
+        :param device_data: NFD
+        :type device_data: dict
+
+        :returns: None -- On Failure
+        :returns: dict -- It has the following scheme
+        {
+            'config': [
+                {
+                    'resource': 'interfaces',
+                    'resource_data': {
+                        ...
+                    }
+                },
+                {
+                    'resource': 'routes',
+                    'resource_data': {
+                        ...
+                    }
+                }
+            ]
+        }
+
+        :raises: exceptions.IncompleteData
+        """
+
+        mgmt_ip = device_data.get('mgmt_ip', None)
+        provider_ip = device_data.get('provider_ip', None)
+        provider_mac = device_data.get('provider_mac', None)
+        provider_cidr = device_data.get('provider_cidr', None)
+        provider_gateway_ip = device_data.get('provider_gateway_ip', None)
+        consumer_ip = device_data.get('consumer_ip', None)
+        consumer_mac = device_data.get('consumer_mac', None)
+        consumer_cidr = device_data.get('consumer_cidr', None)
+        consumer_gateway_ip = device_data.get('consumer_gateway_ip', None)
```

Comment:
In general, get will always return None if key not exists, no need to do explicitly.